lib/ts/ui/index.tsx

```diff
@@ -85,7 +85,20 @@ export default class UI {
     }
 
     static getRoutingComponent(preBuiltUiClassList: PreBuiltRecipes): JSX.Element {
-        const recipeList = preBuiltUiClassList.map((r) => r.getInstanceOrInitAndGetInstance());
+        let recipeList: RecipeRouter[];
+
+        try {
+            recipeList = preBuiltUiClassList.map((r) => r.getInstanceOrInitAndGetInstance());
+        } catch (e) {
+            // TODO during review: Is this checking required here if all recipes internally handle it?
```

Comment:
If `RoutingComponent` can handle this (and I think it should be fine) then there is no need to do this here.